src/mon/AuthMonitor.cc

```diff
@@ -1933,6 +1934,46 @@ bool AuthMonitor::prepare_command(MonOpRequestRef op)
   return false;
 }
 
+
+int AuthMonitor::_is_perm_in_mdscaps_correct(const string& mdscaps,
```

Comment:
Can't such a check be a part of MDSAuthCaps::parse()? MDSAuthCaps::parse() errors out when the cap is invalid (say just `w`) and it should be able to throw out the appropriate error message.

Reply:
I thought about this while writing the code but forgot to pursue this idea after writing the code.

Reply:
Done.